projects.css

```
@@ -0,0 +1,269 @@
+.cs-card{
+  position:relative;
+  width:min(440px, 100%);
+  background:var(--white);
+  border:3px solid var(--ink);
+  box-shadow:var(--shadow);
+  padding:34px 26px 26px;
+  text-align:center;
+  transform:rotate(-1deg);
+  animation:csPop .28s cubic-bezier(.2,1.4,.5,1);
+}
+@keyframes csPop{
```

Stylelint (17.14.0)

[error] Expected keyframe name "csPop" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)